package.json

```diff
@@ -2133,19 +2139,28 @@
         "vscode:prepublish": "webpack --mode production",
         "webpack": "webpack --mode development",
         "webpack-dev": "webpack --mode development --watch",
-        "test-compile": "tsc -p ./"
+        "test-compile": "tsc -p ./",
+        "prepare": "husky"
     },
     "devDependencies": {
+        "@johnnymorganz/stylua-bin": "^0.20.0",
```

Comment:
I don't _love_ this dependency, because it ultimately just fetches a binary from Github, but I think it's at least an ok compromise to ensure the version is managed.

Reply:
hmm. but the scripts are still using `npx`, is that intentional?

Reply:
Correct. It still runs via node, but it's just a thin wrapper to run the binary. I figure this extra layer of dependency actually helps us ensure everyone's running the same stylua version, which is probably a good thing. I wish it weren't just pulling it from Github, though.

https://github.com/JohnnyMorganz/StyLua/blob/049bb7c2c5ff0f764eec8ab3627e114ff23ddabf/stylua-npm-bin/binary.js#L123-L136

Reply:
If we are using `npx`, then I don't think the `devDependencies` entry is (technically) needed, is it?

If we want this entry for "explicitness", that seems fine. But does `npx` actually look at this?

Reply:
Oh, I used `npx` beccause `stylua-bin` didn't run anything, but going back, I realize now that npm handles just `stylua` just fine. :facepalm: 

Will replace